specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -164,6 +166,190 @@ Consider at least the below topics but you do not need to cover those that are n
+
+https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes
+
+#### User Journeys
+
+**Active development**
+
+The VDK frontend empowers users to iterate quickly during the active development
+phase of new data jobs. It provides the necessary tools for developers to view
+the status and output of their jobs, as well as performance and resource data to
+help with optimization.
+
+*Example scenario 1*
+
+<!-- User creates a new job and tests it on different types of input data-->
+A user is actively developing a new data job that should be able to work with
+different data sources. They create and deploy the job using the vdk cli. The
+user then changes the data source configuration and uses the executions tab to
+view logs and check how each data source impacts job duration.
+
+*Example scenario 2*
+<!-- User has data heavy ingestion and wants to optimize their data jobs -->
+A user would like to optimize an in-development data job that uses a complex SQL
+query to ingest a large amount of data. The user has several ideas about
+optimizing the query, but is not sure which approach scales better for a
+sufficiently large amount of data. The user tests all the approaches they have
+in mind by doing code chagnes and creating consecutive data job deployments.
+They use the job duration graph on the data jobs executions details page to
+compare how their approaches scale for a sufficiently large amount of data. The
+user chooses the best approach based on their findings.
+
+*Example scenario 3*
+<!-- User creates a new job, but it fails to start -->
+A user is actively developing a new data job. They do a code change and they see
+their job has failed in the data job details page. They open the logs from the
+provided link and quickly find out what caused the error.
+
+**Monitoring**
+
+The VDK frontend empowers users to detect problems at a glance. It also provides
+the necessary traceablity to do deeper troubleshooting when necessary.
+
+*Example Scenario 1*
+
+<!-- User finds that their job is failing intermittently -->
+A user has a production job that runs every 30 minutes. One morning, the user
+finds a mix of data job success and failure notifications in their inbox. The
+user goes to the data jobs page, finds their job by name, goes to the job's
+details page and navigates to the executions tab. They find that their job
+started failing sporadically the previous day after 00:01 AM.
+
+They check the logs for the first failure and find that they are getting
+rate-limited by the data source. The user then checks the data source's
+documentation and finds that the data source has introduced new rate limits
+starting the previous day. The user then adjusts their job's frequency
+accordingly.
+
+*Example Scenario 2*
+
+<!-- User finds that their job's duration has increased after March 24th -->
+A user has a production job that fetches and curates data, which is used to
```

Comment:
Avoid using the generic persona term as user. Try to be as specific as reasonably possible. 

Often user scenarios would even start with specific name (it's a fictional name but makes the scenario feel more real) 

`Alex, a data engineer in an ingestion team, has a production job that ...`

`Paul, data analyst, ... etc.`

Reply:
I'll change them to Bob and Alice, as is tradition. 😃